### ISSUE TITLE
[Feature Request] Enable --providers.kubernetesingress

### ISSUE THREAD

**Issue by wi1dcard** [containous/traefik-helm-chart#7]
Adding an option to enable the compatibility to Kubernetes Ingress would be great.

**Comment by sgandon** [comment]
indeed, I tried to deploy this chart and it does not seem to be reacting to Kubernetes ingress resources.

**Comment by kopaygorodsky** [comment]
Agree

**Comment by dduportal** [comment]
This issue should be fixed by #34 , with the following value:

```
helm install --name=traefik --set=additionalArguments="{--providers.kubernetesingress=true}" .
```

**Comment by elthariel** [comment]
I still think we want to provide explicit options for those use cases and keep the additional arguments for edge cases
